app/api/users.py

```diff
@@ -14,17 +14,31 @@ def get_user(username):
     abort(404)
 
 
-@api_bp.route('/edit_profile/<user_id>')
+@api_bp.route('/update_profile/', methods=["GET", "POST"])
 @token_auth.login_required
-def edit_profile():
-    is_sitter = request.get_json()['is_sitter']
-    collection = Sitter if is_sitter else Owner
-    token = request.get_json()['token']
-    u = collection.check_token(token)
-    return jsonify(u.to_dict(include_email=True))
+def update_profile():
+   is_sitter = request.get_json()['is_sitter']
+   collection = Sitter if is_sitter else Owner
+   print(request.get_json()['profile_data'])
```

Comment:
printing again

Reply:
please ignore all prints for the time being